### PR TITLE
Fix SELinux and add feature::api::bind_port support

### PR DESCRIPTION
**Pull request by lbetz**
Add support of SELinux. That mean the required policy package is installed if a selinux_package_name in globals is set and facts.os.selinux.enabled=true.

### CODE REVIEW

**Review by bastelfreak — APPROVED** [review]

**Comment by bastelfreak** [comment]
I removed the enhancement label. Our changelog generator puts PRs in categories based on labels. And it gets confused when it has to put a PR into two categories.

**Comment by SimonHoenscheid** [comment]
I would prefer if we keep the "manage_selinux" switch, so the user has to make a decision and it reduces the impact of the change.

What happens if the user runs icinga2 on a different port, with a diffent user etc.?

**Comment by SimonHoenscheid** [comment]
Is there a reason to default manage_SELinux to true?

File: manifests/init.pp [hunk 1]
```diff
@@ -118,7 +118,7 @@
   Boolean                    $enable          = true,
   Boolean                    $manage_repos    = false,
   Boolean                    $manage_packages = true,
-  Boolean                    $manage_selinux  = false,
+  Boolean                    $manage_selinux  = true,
```
Comment (SimonHoenscheid) [inline manifests/init.pp]: This switch only makes sense in my opinion if it defaults to false

**Review by SimonHoenscheid — CHANGES_REQUESTED** [review]

File: manifests/init.pp (hunk 1 above)
Comment (lbetz) [reply manifests/init.pp]: Why? What is the difference between deliberately switching selinux on or off?
By default it's on if selinux is enabled. On the other hand if you wanna disable selinux support if it's on, set manage_selinux to false.

**Review by SimonHoenscheid — APPROVED** [review]

**Comment by SimonHoenscheid** [comment]
@lbetz pointed out in a private chat, this is still a draft.

**Comment by lbetz** [comment]
Now this PR only fixes problems with selinux on some file permissions and add support to change the bind_port of class icinga2::feature::api.

**Review by SimonHoenscheid — APPROVED** [review]

**Pull request merged**